src/Symfony/Component/Yaml/Inline.php

```diff
@@ -51,7 +51,7 @@ public static function initialize(int $flags, int $parsedLineNumber = null, stri
      * Converts a YAML string to a PHP value.
      *
      * @param string $value      A YAML string
-     * @param int    $flags      A bit field of PARSE_* constants to customize the YAML parser behavior
+     * @param int    $flags      A bit field of Yaml::PARSE_* constants to customize the YAML parser behavior
```

Comment:
There are some more similar references in the Yaml component. We should fix them all.

Reply:
I updated other references